tests/.eslintrc

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
@dafortin Do you know why this file will show up here? I can't find this file in my local copy.